docs/sources/write/_index.md

```diff
@@ -1,29 +1,31 @@
 ---
-title: Write documentation
-menuTitle: Write
-description: Learn about Grafana Labs' technical documentation writing guidelines.
-weight: 400
 aliases:
   - /docs/writers-toolkit/writing-guide/
   - /docs/writers-toolkit/write/
-Keywords:
+date: 2024-02-29
+description: Learn about Grafana Labs' technical documentation writing guidelines.
+keywords:
   - writing
   - guidelines
+menuTitle: Write
+title: Write documentation
+weight: 400
 ---
 
 # Write documentation
 
-This writing guide defines the structured authoring environment we use to create documentation at Grafana Labs.
+This writing guide defines how Grafana Labs writes documentation.
 If you write technical documentation for Grafana Labs, familiarize yourself with these guidelines.
 
-We write our documentation using [U.S. English](https://developers.google.com/style/translation).
+## Before you write
 
-The guidelines are for anyone who is interested in improving Grafana Labs' technical content. They are intended to guide you on your documentation journey, whether you are requesting a change, editing a topic, or writing a set of documentation for a new product or feature from scratch.
+To learn how to structure documentation, refer to [Structure](https://grafana.com/docs/writers-toolkit/structure/).
 
-If you are already familiar with the guidelines, feel free to get started from one of the [templates](https://github.com/grafana/writers-toolkit/tree/main/docs/static/templates).
+For guidelines for writing documentation, refer to the following sections:
 
-We hope you find what you are looking for. If you don't, provide us with the feedback, so we can continuously improve this writing guide.
```

Comment:
That's a reasonable question, I removed it because:

1. The rewrite focuses on directing users to more specific and actionable content.
1. It's already present in the [Writers' Toolkit home page](https://grafana.com/docs/writers-toolkit/#questions-and-feedback).
1. Every page also has a section dedicated to feedback.
   <img width="764" alt="Screenshot 2024-03-01 at 08 26 49" src="https://github.com/grafana/writers-toolkit/assets/4599384/d4574f89-102a-49b4-bb05-a0f73f9cd0a1">
1. It doesn't actually  explain how to provide us with feedback anyway.

Reply:
Maybe we should make the word "feedback" a link to the docs email?

I keep meaning to complain about the text we use for those three links at the bottom of every page.  The first one is the only one that makes any sense.  "Report a problem" really should link to opening a GitHub issue, and the link to the department email should say something like "Contact the docs team".

Reply:
I like Julie's suggestion above - if we would like to ask for feedback more directly than the three links at the close of the page, let's tell folks how to do so.